docs/index.html

```diff
@@ -341,6 +341,59 @@ <h2>Terminology</h2>
           <dfn data-lt="URI|URIs">URI</dfn>
         </dt>
         <dd>An identifier as defined by [[RFC3986]].</dd>
+        <dt>
+          <dfn data-lt="WORKFLOW|WORKFLOWs">WORKFLOW</dfn>
+        </dt>
+        <dd>
+          A distinct path for a distinct set of items being traced that either is declared
+          in advance by a user or system, or a root path that has already occurred
+          (in whole or in part). A WORKFLOW might branch, combine with others, or
+          terminate prematurely. It is an abstraction that in a traceability
+          context is inherently directed, and has a declared origin node,
+          and desired end node, with each node being a change in state or location.
+          Properties may be attached to nodes via credentials and correlated to one
+          another (and as a result, to the set of items) through use of a distinct
+          WORKFLOW CORRELATION ID.
+
+          In a real world context, a workflow might be created to describe one party
+          requesting a set of goods from some other location.  The workflow can then
+          be used to group and associate the nodes as those goods move from the origination
+          point (a node) to the end destination.  Any change in state, location, or 
+          control (legal or physical responsibility of some kind for the goods) would be
+          described as a node represented by a VC which has the workflow CORRELATION ID
+          attached to it. 
+        </dd>
+        <dt>
+          <dfn data-lt="CORRELATION ID|CORRELATION IDs|CORRELATIONID|CORRELATIONIDs">CORRELATION ID</dfn>
+        </dt>
+        <dd>
+          A unique identifier assigned to a WORKFLOW complying with [[RFC4122]].
+          A CORRELATION ID MUST conform with UUID Version 4.  
+          Such an identifier is usually created upon the creation of the first
+          node to be associated with a workflow, or upon the creation of a workflow
+          (in whole or in part) after the fact.  This CORRELATION ID is attached to a
+          VC or other credentials by use of a property in the "credentialSubject" 
+          with the property name "correlationId".
+          <aside class="example" title="CORRELATIONID">
+            <p>
+              Use of a CORRELATION ID in a VC
+            <p>
+            <pre class="js">
+              {
+                "@context": [
+                  "https://www.w3.org/2018/credentials/v1",
+                  "https://www.w3.org/2018/credentials/examples/v1"
+                ],
+                "type": ["VerifiableCredential"],
+                "issuer": "did:example:Wz4eUg7SetGfaUVCn8U9d62oDYrUJLuUtcy619",
+                "credentialSubject": {
+                  "workFlowOriginator": "Some User or System",
+                  "product": "Blueberries",
+                  "correlationId": "d2dfc100-9384-49e4-b547-5e7d84ec5bf5"
```

Comment:
this could go outside of the subject... and probably should since its a correlation for the credential